wandb/sdk/data_types/object_3d.py

```diff
@@ -76,6 +79,125 @@ class Camera(TypedDict):
         target: Sequence[Point3D]
 
 
+def _install_numpy_error() -> "wandb.Error":
```

Comment:
Why not just define an error that subclasses our wandb Error instead of a function?

Reply:
mainly because we don't throw a more specific error in other places where `NumPy` is required. In terms of the code it's equivalent, with this being slightly shorter than defining a subclass.

Reply:
you don't like
```python
np = util.get_module("numpy", required="I NEED NUMPY")
```
?